signingvalidation.proj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This **file** is named `signingvalidation.proj`, there's also a `tools-local\signingvalidation` **folder**.  The reason for this was because I wanted an entry point, and then a separate file which the entry point can `MSBuild` after it did its "prerequisites" stuff.  There's no reason I couldn't move `sign.proj` and `signingvalidation.proj` into the same `signing` folder as `sign.proj` and `validate.proj` though..